automation/validate-pvc-name-stability.py

```diff
@@ -0,0 +1,94 @@
+#! python
```

Comment:
The shebang should specify an absolute path. It is recommended to use `env` to locate python for portability. eg: `#!/usr/bin/env python`